index.js

```diff
@@ -185,7 +189,23 @@ var jwt = require('jsonwebtoken');
             // mix the two, updates to device are pushed to cloud and vice versa
 
           }
-
+          
+          //App uninstalls
+          Matrix.service.firebase.app.watchUserAppsRemoval(function (app) {
```

Comment:
We should think of a process around making sure to let others know about updated repo modules. Maybe in the PR description? I needed to update firebase.